js/ui/number_box/number_box.mask.js

```diff
@@ -542,6 +569,13 @@ var NumberBoxMask = NumberBoxBase.inherit({
         return this.callBase.apply(this, arguments);
     },
 
+    _removeStubInText: function(text) {
+        var decimalSeparator = number.getDecimalSeparator(),
+            regExp = new RegExp("[^0-9" + decimalSeparator + "]+", "g");
```

Comment:
Should we do that if `text` is defined?

Reply:
Text cannot be undefined because it is always input().val()

Reply:
```
 return (text || this._input().val()).replace(regExp, "");
```
long expression confused me, I've missed the first closing bracket. Can you make it more readable? :)